cdap-ui/app/cdap/components/AbstractWidget/FormInputs/MultiSelect/index.tsx

```diff
@@ -14,12 +14,12 @@
  * the License.
  */
 
-import React, { useState, useEffect } from 'react';
-import Select from '@material-ui/core/Select';
 import MenuItem from '@material-ui/core/MenuItem';
+import Select from '@material-ui/core/Select';
```

Comment:
nit: Did you accidentally change the order of the imports?

Reply:
This is probably okay, since it has been handled by VS code's `"editor.codeActionsOnSave": {"source.organizeImports": true}`!